src/main/java/com/conveyal/gtfs/api/graphql/RouteFetcher.java

```diff
@@ -44,6 +45,27 @@
         return routes;
     }
 
+    public static List<WrappedGTFSEntity<Route>> fromStop(DataFetchingEnvironment environment) {
+        WrappedGTFSEntity<Stop> stop = (WrappedGTFSEntity<Stop>) environment.getSource();
+        List<String> routeIds = environment.getArgument("route_id");
```

Comment:
If you already know what routes stop at a stop, do you need to use this function?